examples/scale-up/kubernetes_orignal_params.json

```diff
@@ -38,12 +38,21 @@
   "firstConsecutiveStaticIP": {
     "value": "10.240.255.5"
   },
+  "kubeClusterCidr": {
```

Comment:
Would it be better to set the defaults like this so that you don't need to touch existing templates.

https://github.com/Azure/acs-engine/blob/master/parts/kubernetesparams.t#L66